classes/class-uabb-cloud-templates.php

```diff
@@ -1,25 +1,46 @@
 <?php
-
 /**
  * UABB Cloud Templates initial setup
  *
  * @since 1.2.0.2
+ * @package UABB Cloud Templates
  */
 
-if( !class_exists('UABB_Cloud_Templates') ) {
-
+if ( ! class_exists( 'UABB_Cloud_Templates' ) ) {
+	/**
+	 * This class initializes UABB Cloud Templates
+	 *
+	 * @class UABB_Cloud_Templates
+	 */
 	class UABB_Cloud_Templates {
-
+		/**
+		 * Holds an instance of Cloud Templates.
+		 *
+		 * @since x.x.x
```

Comment:
update version number from x.x.x to the specific version when this class or function was introduced in the entire file